### DISCUSSION TITLE
What is 'Use QuantMatMul'?

### DISCUSSION THREAD

**Discussion by StripedPuppy** [LostRuins/koboldcpp#361] [General]
I can't seem to find documentation anywhere on the net. I'm just not sure if I should mess with it or not.

Preset: CuBLAS
GPU: Nvidia RTX-3060
CPU: Intel i7-12700
Model: Mostly 7b models at 8_0 quant

**Comment by LostRuins** [comment]
I'll add it to the wiki soon. It's basically a new approach for prompt processing from upstream.
